lib/galaxy/tools/deps/commands.py

```diff
@@ -70,7 +70,7 @@ def execute(cmds):
 
     Return the standard output if the commands are successful
     """
-    return _wait(cmds, shell=False)
+    return __wait(cmds, shell=False, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
```

Comment:
Change back `__wait()` to `_wait()` ? You haven't changed the function name below and double underscore should be used only if you need name mangling (see https://stackoverflow.com/questions/1301346/the-meaning-of-a-single-and-a-double-underscore-before-an-object-name-in-python ).